### ISSUE TITLE
Electrum server didnt provide txhex to properly create PSBT transaction

### ISSUE THREAD

**Issue by cast4way** [BlueWallet/BlueWallet#3097]
I get the following error if I connect BW with remote electrum servers (tried 2) when I want to send a transaction from a multisig vault:

Error  
Electrum server didnt provide txhex to properly create PSBT transaction

Using the default server workes fine.

**Comment by Overtorment** [comment]
yes, thats the error thrown inside multisig wallet if for some tx input there is no txhex.
that might just be connectivity issue (which by coincidence happened when you tried some random public servers).
also, which version did you try? there were some improvements made around that area in v6.1.0

**Comment by cast4way** [comment]
I am on v6.1.0, I used my umbrel electrum server (that seems pretty slow with BW in general) and a random public one. I agree, connectivity issue sounds plausible.

**Comment by Overtorment** [comment]
which one had issues?
have you tried several times?

**Comment by cast4way** [comment]
I tried my umbrel and two random public electrum servers multiple times, without success.
I have this issue with an all air gapped multisig 2/3.
This issue does not occur on my all BW multisig 2/3.

**Comment by Overtorment** [comment]
can you share coordination setup privately to i@bluewallet.io ? ill try to reproduce.
my working theory is that your inputs are thick transactions (like batched withdrawals from exchanges) and public electrums have lower limits on the size of the response and thus cannot return  full transaction hex

**Comment by Overtorment** [comment]
is it still happening?
